rasa/validator.py

```diff
@@ -416,19 +416,16 @@ def verify_domain_validity(self) -> bool:
         return True
 
     def verify_config_mandatory_keys(self) -> bool:
```

Comment:
Maybe we should rename this to something like: warn_if_assistant_id_is_not_set ?
And we should remove return value since it is always true and it does not change at all.